gammapy/image/utils.py

```diff
@@ -137,11 +148,22 @@ def ring_correlate(image, r_in, r_out, mode='constant'):
 
     Parameters
     ----------
-    TODO
+    image : array_like
+        Image to be correlated.
+
+    r_in : float
+        Ring inner radius in pixels.
+
+    r_out : float
+        Ring outer radius in pixels.
+
+    mode : {‘reflect’,’constant’,’nearest’,’mirror’, ‘wrap’}, optional
+	the mode parameter determines how the array borders are handled. For ‘constant’ mode, values beyond borders are set to be cval. Default is ‘constant’.
```

Comment:
Try to roughly respect the 80 character line length limit so that everything is readable in the code editor without scrolling left and right. Here you should start with "For ..." on a new line.